docker/compose/Readme.md

```diff
@@ -7,16 +7,52 @@ You may obtain a copy of the License at
 
     http://www.apache.org/licenses/LICENSE-2.0
 -->
-# docker-compose
+# Containerized Deployment Using Docker Compose
 
 Docker compose can be used to quickly spin up a Pravega cluster to use for testing or development. Unlike 
-`pravega-standalone`, a compose cluster will use a real standalone HDFS, Zookeeper, BookKeeper, and will run the 
-Segment Store and the Controller in separate processes.
+`pravega-standalone`, a Compose cluster will use a real standalone HDFS, Zookeeper and BookKeeper, and will also run
+Segment Store and Controller in separate processes.
 
 ## Running
 
-The IP or hostname must be provided as a `HOST_IP` environment variable. Otherwise, things should just work.
+1. Ensure that you have Docker and Docker Compose installed.
```

Comment:
@ravisharda 1.Ensure that  Docker and Docker Compose installed.

Reply:
Fixed.